mobile/src/main/java/org/hawkular/client/android/push/PushClient.java

```diff
@@ -56,7 +59,14 @@ public void setUpPush() {
             return;
         }
 
-        RegistrarManager.config(PushConfiguration.NAME, AeroGearGCMPushConfiguration.class)
+        FirebaseApp.initializeApp(this.context, new FirebaseOptions.Builder()
```

Comment:
Do we really need this? I'm asking because I wanna move the properties values to the json like this one https://github.com/aerogear/aerogear-android-cookbook/blob/master/HelloPush/app/src/main/assets/push-config.json